dataedit/views.py

```diff
@@ -917,6 +917,7 @@ def get(self, request, schema, table):
                 current_view = default
 
         table_views = list(chain((default,), table_views))
+        title_name = metadata["title"]
```

Comment:
```suggestion
        title_name = metadata.get("title")
```
This does not throw an exception if the "title" field is missing

Reply:
Review Comment addressed.